src/Data/Uniques/sword.lua

```diff
@@ -340,15 +324,17 @@ Adds 2 to 6 Physical Damage
 Dreadbeak
 Rusted Sword
 Source: Drops from any endgame map boss
+Variant: Pre 2.6.0
 Variant: Pre 3.7.0
 Variant: Current
-Requires Level 61
+LevelReq: 61
 Implicits: 1
```

Comment:
Needs to be 2 for second implicit variant:
```suggestion
Implicits: 2
```